crates/ark/src/fixtures/dummy_frontend.rs

```diff
@@ -170,6 +179,35 @@ impl DerefMut for DummyArkFrontendNotebook {
     }
 }
 
+impl DummyArkFrontendDefaultRepos {
```

Comment:
Nice to see this infra put to good use. I wish we could make it more ergonomic but it's hard to ensure proper configuration since tests are run in nondeterministic order.

Reply:
Yeah, I was delighted to see that there's a way to accomplish this in tests!